### PR TITLE
fix edit on github button url

### LINKED ISSUE #459: edit on github link broken for manifest reference page
The `edit on github` button at the bottom left of the [manifest reference](https://developer.fermyon.com/spin/manifest-reference.md) document seems to be broken. It is redirecting me to a link with an extra `.md` at the end: https://github.com/fermyon/developer/blob/main/content/spin/manifest-reference.md.md

### PR DESCRIPTION
**Pull request by karthik2804**
fixes #459

### CODE REVIEW

File: content/spin/build.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ template = "spin_main"
 date = "2022-03-14T00:22:56Z"
 enable_shortcodes = true
 [extra]
-url = "https://github.com/fermyon/spin/blob/main/docs/content/build.md"
+url = "https://github.com/fermyon/developer/blob/main//content/spin/build.md"
```
Comment (itowlson) [inline content/spin/build.md:6]: Double slash?

**Review by itowlson — COMMENTED** [review]
I'm not able to review this, sorry.  I don't understand the impact of changing the `url` in the metadata; I thought that was meant to point to the page on the site, not to the GH source, and I don't know whether it's correct to change it, or how it relates to the HTML changes.  Sorry.

**Review by tpmccallum — COMMENTED** [review]
Hi @karthik2804 
Noticed a bunch of consecutive forward slashes. Not sure if intentional, just mentioning (used a command to create output below with file names for convenience).

```bash
find content -type f -exec grep -H '[a-z]\/\/' {} \;
```

```bash
content/bartholomew/contributing-bartholomew.md:url = "https://github.com/fermyon/developer/blob/main//content/bartholomew/contributing-bartholomew.md"
content/bartholomew/markdown.md:url = "https://github.com/fermyon/developer/blob/main//content/bartholomew/markdown.md"
content/bartholomew/templates.md:url = "https://github.com/fermyon/developer/blob/main//content/bartholomew/templates.md"
content/bartholomew/quickstart.md:url = "https://github.com/fermyon/developer/blob/main//content/bartholomew/quickstart.md"
content/bartholomew/seo.md:url = "https://github.com/fermyon/developer/blob/main//content/bartholomew/seo.md"
content/bartholomew/index.md:url = "https://github.com/fermyon/developer/blob/main//content/bartholomew/index.md"
content/bartholomew/configuration.md:url = "https://github.com/fermyon/developer/blob/main//content/bartholomew/configuration.md"
content/bartholomew/themes.md:url = "https://github.com/fermyon/developer/blob/main//content/bartholomew/themes.md"
content/bartholomew/shortcodes.md:url = "https://github.com/fermyon/developer/blob/main//content/bartholomew/shortcodes.md"
content/bartholomew/scripting.md:url = "https://github.com/fermyon/developer/blob/main//content/bartholomew/scripting.md"
content/spin/distributing-apps.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/distributing-apps.md"
content/spin/javascript-components.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/javascript-components.md"
content/spin/architecture.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/architecture.md"
content/spin/contributing-spin.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/contributing-spin.md"
content/spin/deploying-to-fermyon.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/deploying-to-fermyon.md"
content/spin/plugin-authoring.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/plugin-authoring.md"
content/spin/url-shortener.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/url-shortener.md"
content/spin/quickstart.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/quickstart.md"
content/spin/go-components.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/go-components.md"
content/spin/running-apps.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/running-apps.md"
content/spin/install.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/install.md"
content/spin/redis-trigger.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/redis-trigger.md"
content/spin/dynamic-configuration.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/dynamic-configuration.md"
content/spin/other-languages.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/other-languages.md"
content/spin/rdbms-storage.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/rdbms-storage.md"
content/spin/index.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/index.md"
content/spin/template-authoring.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/template-authoring.md"
content/spin/python-components.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/python-components.md"
content/spin/build.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/build.md"
content/spin/manifest-reference.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/manifest-reference.md"
content/spin/redis-outbound.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/redis-outbound.md"
content/spin/extending-and-embedding.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/extending-and-embedding.md"
content/spin/kv-store.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/kv-store.md"
content/spin/rust-components.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/rust-components.md"
content/spin/http-trigger.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/http-trigger.md"
content/spin/spin-oci.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/spin-oci.md"
content/spin/http-outbound.md:url = "https://github.com/fermyon/developer/blob/main//content/spin/http-outbound.md"
content/cloud/user-settings.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/user-settings.md"
content/cloud/deployment-concepts.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/deployment-concepts.md"
content/cloud/quickstart.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/quickstart.md"
content/cloud/develop.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/develop.md"
content/cloud/faq.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/faq.md"
content/cloud/fermyon-cloud.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/fermyon-cloud.md"
content/cloud/data-postgres.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/data-postgres.md"
content/cloud/deploy.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/deploy.md"
content/cloud/support.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/support.md"
content/cloud/index.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/index.md"
content/cloud/delete.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/delete.md"
content/cloud/data-redis.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/data-redis.md"
content/cloud/spin-vs-code-extension.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/spin-vs-code-extension.md"
content/cloud/upgrade.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/upgrade.md"
content/cloud/rest-api.md:url = "https://github.com/fermyon/developer/blob/main//content/cloud/rest-api.md"
```

File: content/spin/build.md (hunk 1 above)
Comment (karthik2804) [reply content/spin/build.md:6]: good catch!

**Comment by karthik2804** [comment]
I will fix the double-forward slash soon!

File: content/spin/build.md (hunk 1 above)
Comment (karthik2804) [reply content/spin/build.md:6]: That URL is just embedded in the `edit on github` button. The `extra.url` is used nowhere else.

**Comment by karthik2804** [comment]
@tpmccallum updates the links. ready for another review.

**Review by tpmccallum — APPROVED** [review]

**Pull request merged**